### PR TITLE
[frontend] fix broken Delete button in Analyst Workbench interface (#3363)

### LINKED ISSUE #3363: Broken Delete button in Analyst Workbench interface
## Description

The "inline" Delete Button to delete an entity, an observable, a container or a relationship in the analyst Workbench interface is broken. Nothing happen when it is clicked, even if the related checkbox is checked. 
image

To note: the Delete button in the toolbar is working fine.

## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: { e.g. OpenCTI 1.0.2 }
3. OpenCTI client: { e.g. frontend or python }
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. { e.g. Run ... }
2. { e.g. Click ... }
3. { e.g. Error ... }

### PR DESCRIPTION
**Pull request by SouadHadjiat**
### Proposed changes

* fix call to function onClick (the argument was the event object)
* minor fix for dialog open attribute which should be a boolean (errors in console)

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/3363

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js [hunk 1]
```diff
@@ -3686,7 +3686,7 @@ const WorkbenchFileContentComponent = ({
       {currentTab === 2 && renderRelationships()}
       {currentTab === 3 && renderContainers()}
       <Dialog
-        open={deleteObject}
+        open={!!deleteObject}
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js:3689]: Why is that necessary ? It seems that deleteobject is either null, undefined or an object ?

Comment (SouadHadjiat) [reply opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js:3689]: `open` parameter should be a boolean, I had warnings in the console

Comment (Kedae) [reply opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js:3689]: Oh, I see. In that case we might want to enforce a rule between this usage and the Boolean(deleteObject) one ? What do you like the most ?

**Comment by Archidoit** [comment]
Behavior tested --> OK

**Review by marieflorescontact — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js (hunk 1 above)
Comment (SouadHadjiat) [reply opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js:3689]: a rule ? I don't understand

**Pull request merged**